provider/testdata/Cargo.toml

```diff
@@ -98,6 +98,12 @@ locales = [
     # - Interesting case-mappings
     "tr",
 
+    # Chinese:
+    # - Multiple scripts
+    # - Complex date patterns
+    "zh",
+    "zh-Hant",
```

Comment:
acknowledged.  I added it in order to test the new `B` behavior, which is default in zh-Hant.  But I'll add a one-off test case for that instead of pulling in the whole locale.

Reply:
done.